docs/pages/eas-metadata/getting-started.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Getting started with EAS Metadata
+sidebar_title: Getting started
+---
+
+import { Terminal } from '~/ui/components/Snippet';
+
+> ⚠️ EAS Metadata is in beta and subject to breaking changes.
+
+EAS Metadata helps you prepare your app for review in the app stores and helps you prevent common pitfalls that often could lead to an app rejection.
+
+## Prerequisites
+
+EAS Metadata is available starting from EAS CLI >= 0.54.0, and _only supports the App Store_.
+
+## Create a local store configuration
+
+We first need to create our local **store.config.json** to get started. This file keeps track of all the information for the app stores. If you have an existing app in the stores, you can generate the configuration file by running:
+
+<Terminal cmd={['$ eas metadata:pull']} />
+
+If you don't have an app in the stores, you can create the **store.config.json** manually.
+
+```json
+{
+  "configVersion": 0,
+  "apple": {
+    "info": {
+      "en-US": {
+        "title": "Awesome App",
+        "subtitle": "Your self-made awesome app",
+        "description": "The most awesome app you've ever seen",
+        "keywords": ["awesome", "app"],
+        "marketingUrl": "https://example.com/en/promo",
+        "supportUrl": "https://example.com/en/support",
+        "privacyPolicyUrl": "https://example.com/en/privacy"
+      }
+    }
+  }
+}
+```
+
+> By default EAS Metadata will use the **store.config.json** file in your project. You can change the name and location of the file by [configuring the `metadataPath` on the EAS Submit](../submit/eas-json.md#metadatapath) profile.
+
+## Update the local store configuration
+
+Now it's time to edit the **store.config.json** file and customize it to your app needs. You can find all available options for the **store.config.json** in the [store configuration reference](./store-json.md).
+
+## Upload a new version of the app
+
+To create a new app version in the stores, you must upload a new binary. We can do this by running:
+
+<Terminal cmd={[
+  '# Build and deploy to the stores',
+  '$ eas build --auto-submit',
+  '# Or deploy a previous build to the stores',
+  '$ eas submit',
+]} />
```

Comment:
ideally we can leave this terminal bit out and just link to some other docs page that tells people about getting their app on the store, or assume that they know how to do this already

Reply:
testing a new tool

###### <a href="https://getunblocked.com"><img src="https://avatars.githubusercontent.com/in/166219" width=20 align="center" /></a>&nbsp;&nbsp;Comment posted from [Unblocked](https://getunblocked.com/dashboard/team/5bf29d2c-23ae-4af8-8ccc-fe767cd2df79/thread/ef2c770e-87b7-4d63-8858-dc70e3c51f4a?message=611dfd65-7825-4d07-8daa-61f09335b64e).

Reply:
We can link to this if that makes more sense? https://docs.expo.dev/distribution/uploading-apps/

Reply:
that works!

--

Comment posted using [Unblocked](https://getunblocked.com/dashboard/team/5bf29d2c-23ae-4af8-8ccc-fe767cd2df79/thread/ef2c770e-87b7-4d63-8858-dc70e3c51f4a?message=42fca110-5457-4386-9424-5f204e7168c4).

Reply:
Done!